### PR TITLE
Merge 2.0-preview branch into 2.0-analyzer.

### PR DESCRIPTION
**Pull request by aomarks**
This brings in Elliott's 2.0-preview changes to the older elements and fixes up bower.json to get Travis green again.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,20 +3,26 @@ sudo: required
 before_script:
   - npm install -g bower polymer-cli@next
   - polymer install --variants
+  - sudo mv /usr/bin/google-chrome /usr/bin/google-chrome-old
+  - sudo mv /usr/bin/google-chrome-beta /usr/bin/google-chrome
```
Comment (samuelli) [inline .travis.yml]: Line 6 & 7 should be removed, 4 needs to be updated per #119

File: .travis.yml [hunk 2]
```diff
@@ -3,20 +3,26 @@ sudo: required
...
 env:
   global:
-    - secure: JibMotHxRDu0L2Fe4raETzvCmvO1CQ0gG7O0XhfGArhuwTC+IW7MLlO/wYpVUkfCLoSKDvIiLSYKpPtUPJ+kZy04VENNc/axKcPdr4iu54se+qe61mRmAunMBVrS21SKlxnqflO1iPFyJ71IyD0/hGBY76Chj4AMUDWhrHJ0WjE=
-    - secure: qEULPMDsjwPoNkedrd9azBPPTxckC5hWGKAD8hceTOhoFTxNKKkQmB/Hpx+DomAWXq9yFflB9nRqSTnw9la+Py2bHPWohD9tz+YVsZPvhJmTugrOL+i8B7IzCtncCm7WfWDyjxQFYaHg+UIiEcg9Ci29sNAsWnKakl82e1FJ8oU=
+    - secure: >-
+        JibMotHxRDu0L2Fe4raETzvCmvO1CQ0gG7O0XhfGArhuwTC+IW7MLlO/wYpVUkfCLoSKDvIiLSYKpPtUPJ+kZy04VENNc/axKcPdr4iu54se+qe61mRmAunMBVrS21SKlxnqflO1iPFyJ71IyD0/hGBY76Chj4AMUDWhrHJ0WjE=
+    - secure: >-
+        qEULPMDsjwPoNkedrd9azBPPTxckC5hWGKAD8hceTOhoFTxNKKkQmB/Hpx+DomAWXq9yFflB9nRqSTnw9la+Py2bHPWohD9tz+YVsZPvhJmTugrOL+i8B7IzCtncCm7WfWDyjxQFYaHg+UIiEcg9Ci29sNAsWnKakl82e1FJ8oU=
```
Comment (samuelli) [inline .travis.yml:11]: These changes look unintentional?

File: .travis.yml [hunk 3]
```diff
@@ -3,20 +3,26 @@ sudo: required
...
 node_js: stable
 addons:
-  firefox: '46.0'
+  firefox: latest
   apt:
     sources:
       - google-chrome
     packages:
-      - google-chrome-stable
+      - google-chrome-beta
```
Comment (samuelli) [inline .travis.yml]: Should stay as stable per #119

File: .travis.yml [hunk 4]
```diff
@@ -3,20 +3,26 @@ sudo: required
...
   sauce_connect: true
 script:
   - xvfb-run polymer test
-  - 'if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer test -s ''default''; fi'
+  - >-
```
Comment (samuelli) [inline .travis.yml:23]: Unintentional?

File: iron-doc-property-styles.html [hunk 5]
```diff
@@ -94,13 +94,13 @@
...
       #type {
-        @apply(--paper-font-code1);
+        @apply --paper-font-code1;
 
         font-style: italic;
       }
 
       .annotation {
-        color: var(--iron-doc-viewer-property-annotation-color, --paper-grey-700);
+        color: var(--iron-doc-viewer-property-annotation-color, var(--paper-grey-700));
```
Comment (samuelli) [inline iron-doc-property-styles.html:103]: I think these were correct before.

**Review by samuelli — CHANGES_REQUESTED** [review]

File: iron-doc-property-styles.html (hunk 5 above)
Comment (aomarks) [reply iron-doc-property-styles.html:103]: Conforms to "Update custom property syntax" @ https://www.polymer-project.org/2.0/docs/upgrade#shadow-dom-styles

File: .travis.yml (hunk 4 above)
Comment (aomarks) [reply .travis.yml:23]: See above.

File: .travis.yml (hunk 3 above)
Comment (aomarks) [reply .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (aomarks) [reply .travis.yml:11]: This is how the Travis config is formatted on all the 2.0-preview branches, so I figured I'd keep it consistent:

https://github.com/PolymerElements/iron-doc-viewer/blob/2.0-preview/.travis.yml
https://github.com/PolymerElements/paper-button/blob/2.0-preview/.travis.yml

File: .travis.yml (hunk 1 above)
Comment (aomarks) [reply .travis.yml]: Done.

**Review by samuelli — APPROVED** [review]

**Pull request merged**